examples/webhooks-manager/pages/index.tsx

```diff
@@ -159,7 +161,7 @@ export default function Main({
         className="cs1 ce12 form-example--main-content"
         onSubmit={createWebhook}
       >
-        {webhooks.length && <hr />}
+        {boardWebhooks.length && <hr />}
```

Comment:
if length is `0`, this will render the `0`.
```suggestion
        {Boolean(boardWebhooks.length) && <hr />}
```